examples/convert_smartobject_to_layer.py

```diff
@@ -0,0 +1,19 @@
+"""Convert Smart object to artLayer."""
+
+# Import local modules
+from photoshop import Session
+
+
+# example 1
+with Session() as ps:
+    js = """
+var idplacedLayerConvertToLayers = stringIDToTypeID( "placedLayerConvertToLayers" );
+executeAction( idplacedLayerConvertToLayers, undefined, DialogModes.NO );
+"""
+    ps.app.doJavaScript(js)
+
+# example 2
+with Session() as ps:
+    descriptor = ps.ActionDescriptor
+    idplacedLayerConvertToLayers = ps.app.stringIDToTypeID("placedLayerConvertToLayers")
+    ps.app.executeAction(idplacedLayerConvertToLayers, descriptor)
```

Comment:
I tried this in ps6 but it doesn't work. The ScriptListener showed that working js should be:
```
var idnewPlacedLayer = stringIDToTypeID( "newPlacedLayer" );
executeAction( idnewPlacedLayer, undefined, DialogModes.NO );
```
and following py works:
```
import photoshop.api as ps
app = ps.Application()
idnewPlacedLayer = app.stringIDToTypeID("newPlacedLayer")
app.executeAction(idnewPlacedLayer, None)
```
Note: If you replace the `None` with something else, executeAction would fail.

Reply:
`idplacedLayerConvertToLayers` does not even exist in CS6. Evidence:
First, you can app.stringIDToTypeID **ANY** string **without error** and Photoshop will assign ID for them.
So let's restart photoshop and try:
```
>>> import photoshop.api as ps
>>> app = ps.Application()
>>> idnonsense = app.stringIDToTypeID("nonsense")
>>> idnonsense
2520
>>> idplacedLayerConvertToLayers = app.stringIDToTypeID("placedLayerConvertToLayers")
>>> idplacedLayerConvertToLayers
2521
```
`'nonsense'` is obviously not a built-in keyword of Photoshop, so `'placedLayerConvertToLayers'` is neither one. (2521 = 2520 + 1)

So this maybe a different between Photoshop versions, but I still advice to test if this work on later version of ps.